### PR TITLE
Add M. Haseeb in the list of developers

### PR DESCRIPTION
**Pull request by RemiLehe**

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -98,6 +98,11 @@
         "name": "Gu, Junmin",
         "orcid": "0000-0002-1521-8534"
       },
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Haseeb, Muhammad",
+        "orcid": ""
```
Comment (RemiLehe) [inline .zenodo.json]: @mhaseeb123 Could you let us know what your ORCID number is?

Comment (mhaseeb123) [reply .zenodo.json]: https://orcid.org/0000-0002-0697-6894

Comment (mhaseeb123) [inline .zenodo.json]:
```suggestion
        "orcid": "0000-0002-0697-6894"
```

**Review by mhaseeb123 — APPROVED** [review]

File: .zenodo.json (hunk 1 above)
Comment (RemiLehe) [inline .zenodo.json:104]:
```suggestion
        "orcid": "0000-0002-0697-6894"
```

**Pull request merged**